src/views/partition-table/components/Row.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <tr :class="{ error: error }">
+    <td>
+      <input
+        class="input is-size-7-mobile is-size-7-tablet"
+        type="text"
+        placeholder="Name"
+        maxlength="16"
+        v-model="sName"
+      />
+    </td>
+    <td class="w-md">
+      <v-select
+        :options="types"
+        v-model="sType"
+        placeholder="Type"
+        taggable
+        selectOnTab
+      />
+    </td>
+    <td class="w-md">
+      <v-select
+        :options="subtypes"
+        v-model="sSubType"
+        label="label"
+        placeholder="Sub Type"
+        taggable
+        selectOnTab
+      />
+    </td>
+    <td>
+      <input
```

Comment:
Might be worth to add some input mask so user can't enter non-hex characters ?

Reply:
Added the check as in save button because user can enter both number and hex values there and other values which will be translated to hex internally